nimx/private/windows/emscripten_window.nim

```diff
@@ -26,7 +26,7 @@ proc getCanvasDimensions(id: cstring, cssRect: var Rect, virtualSize: var Size)
         setValue($2 + 4, c.height, 'float');
         """, id, addr cssRect, addr virtualSize)
 
-proc eventLocationFromJSEvent(mouseEvent: ptr EmscriptenMouseEvent, w: EmscriptenWindow, eventTargetIsCanvas: bool): Point =
+proc eventLocationFromJSEvent(mouseEvent: ptr EmscriptenMouseEvent | EmscriptenTouchPoint, w: EmscriptenWindow, eventTargetIsCanvas: bool): Point =
```

Comment:
Please move the non-generic code out of this proc to reduce code bloat. E.g.:
```nim
proc eventLocationFromJSEvent(mouseEvent: ptr EmscriptenMouseEvent | EmscriptenTouchPoint, w: EmscriptenWindow, eventTargetIsCanvas: bool): Point =
    eventLocationFromJSCoords(mouseEvent.targetX, mouseEvent.targetY, w, eventTargetIsCanvas)
```
`eventLocationFromJSCoords` should be non-generic.